quests/hello_puppet.md

```diff
@@ -140,46 +140,47 @@ system that can access the master.
 
 <div class = "lvm-task-number"><p>Task 3:</p></div>
 
-To get started, use `ssh` to connect to `hello.puppet.vm`. Your credentials
-are:
-
-**username: learning**  
-**password: puppet**
+To get started, use `ssh` to connect to `hello.puppet.vm` by running:
 
     ssh learning@hello.puppet.vm
 
-Now that you're connected to this system, copy in the command listed below to
-load the agent installer from the master and run it.
+Your credentials are:
```

Comment:
It's surprising how frequently people go to log in and get confused when they're prompted for credentials that they haven't yet seen--they switch windows and don't expect to have to return to the guide until they've completed the given task, so they don't always think to look back and find the login information. Is there a good way to keep these credentials before the command?